src/services/lnd/utils-bos.ts

```diff
@@ -75,13 +85,16 @@ export const rebalancingInternalChannels = async () => {
         destination: selfPubkey,
         ...settings,
       })
+
+      await waitFor({ func: checkNoPendingHtlc, max: 20 })
```

Comment:
why do we need to add this? are we moving tests validations to main code?

Reply:
This was probably poorly named here, did a refactor https://github.com/GaloyMoney/galoy/pull/3016/commits/9300b6b6df4f9d360035cd2205dea010cc1f459c

It's needed because we have a limitation where we can't correctly account for the escrow of a channel if there are pending htlcs in it (see #3031). This is to protect against an edge-case that seems to consistently happen where there are still pending htlcs in our internal channel from the `rebalance` step when cron moves on to the `updateEscrows` step which causes it to consistently record an escrow that is slightly off (by 430 sats).